EPPs/etc/customer_manifest.html

```diff
@@ -5,11 +5,15 @@
 </head>
 <body><div style="font-family:Calibri,Arial, sans-serif; font-size:12.0pt">
     <p> Dear <span style="color: #ff0000"> CUSTOMER NAME(S)</span>,</p>
-    <p>Thank you for choosing to use our sequencing service. I am the LIMS Manager at Edinburgh Genomics Clinical and I will liaise with you from sample submission up until the release of your data. I will reference this project using <b>{{ project }}</b> in the subject line of all future email correspondence.<br>
+    <p><span  style="color: #ff0000">Thank you for choosing to use our sequencing service. I am the Project Coordinator at Edinburgh Genomics Clinical and I will liaise with you from sample submission up until the release of your data. I will reference this project using <b>{{ project }}</b> in the subject line of all future email correspondence.<br>
         <br>Please find attached the following two documents:<br>
+        Thank you for choosing to use our sequencing service again. Please find attached the following two documents:<br>
+        Please find attached the following two documents for your new batch of samples:<br>
+        Please find attached the following two documents for the replacement samples:<br></span>
     <br>1) Sample Manifest &#8208; please complete all of the fields in the green columns that are relevant to your samples and return the file to EGCG-Projects@ed.ac.uk. We can only begin processing when both the manifest and samples have been received at Edinburgh Genomics.<br>
     <br>2)	Sample Submission Requirements &#8208; this document provides guidance on preparing and shipping your samples to Edinburgh Genomics Clinical.<br>
-        <br>Please note that we will require the sample(s) to have a concentration of <b>>=50ng/ul</b> at a volume of <b>>= 50 ul</b>. The sample(s) should only be shipped to us in the barcoded container(s) that we will provide. Plates must be sent frozen on dry-ice but tubes may be sent at ambient temperature.</p>
+        <br>Please note that we will require the sample(s) to have a concentration of <b>>=50ng/ul</b> at a volume of <b>>= 50 ul</b>. The sample(s) should only be shipped to us in the barcoded container(s) that we will provide. Plates must be sent frozen on dry-ice but tubes may be sent at ambient temperature.<br>
+    <br><span  style="color: #ff0000">An order has been placed for the shipment of the sample collection containers to you and these are estimated to arrive on WEEK-DAY DD-MM-YYYY/The container(s) will be placed in your Roslin mailbox by TIME DAY.</span> If you have any questions about samples submission then please don't hesitate to contact me.</p>
```

Comment:
> The container(s) will be placed in your Roslin mailbox by TIME DAY

Same thing - are we assuming customers will always be at Roslin?

Reply:
As above, The new project team has requested a range of text that covers most possible scenarios and they will delete what they don't need.

Reply:
I should add that the project team reviewed the template text prior to the pull request.